src/Elliptic/Systems/ScalarGaussBonnet/Tags.hpp

```diff
@@ -0,0 +1,137 @@
+
+/// \cond
+class DataVector;
+/// \endcond
+
+namespace sgb {
+/// Tags related to solving the scalar equation in sGB gravity.
+namespace OptionTags {
+
+struct Epsilon2 {
+  static std::string name() { return "Epsilon2"; }
+  using type = double;
+  static constexpr Options::String help{
+      "Epsilon2 (quadratic term) for the coupling function."};
+};
+
+struct Epsilon4 {
+  static std::string name() { return "Epsilon4"; }
+  using type = double;
+  static constexpr Options::String help{
+      "Epsilon4 (quartic term) for the coupling function."};
+};
+
+struct RolloffLocation {
+  static std::string name() { return "RolloffLocation"; }
+  using type = double;
+  static constexpr Options::String help{
+      "Location of center of tanh function for rolling off the shift."};
+};
+
+struct RolloffRate {
+  static std::string name() { return "RolloffRate"; }
+  using type = double;
+  static constexpr Options::String help{
+      "Rate of roll off for rolling off the shift."};
+};
+
+}  // namespace OptionTags
+namespace Tags {
+
+/*!
+ * \brief Epsilon2 (quadratic term) for the coupling function.
+ */
+struct Epsilon2 : db::SimpleTag {
+  using type = double;
+  using option_tags = tmpl::list<OptionTags::Epsilon2>;
+  static constexpr bool pass_metavariables = false;
+  static double create_from_options(const double epsilon) { return epsilon; }
+};
+
+/*!
+ * \brief Epsilon4 (quartic term) for the coupling function.
+ */
+struct Epsilon4 : db::SimpleTag {
+  using type = double;
+  using option_tags = tmpl::list<OptionTags::Epsilon4>;
+  static constexpr bool pass_metavariables = false;
+  static double create_from_options(const double epsilon) { return epsilon; }
+};
+
+/*!
+ * \brief Location of center of tanh function for rolling off the shift.
+ */
+struct RolloffLocation : db::SimpleTag {
+  using type = double;
+  using option_tags = tmpl::list<OptionTags::RolloffLocation>;
+  static constexpr bool pass_metavariables = false;
+  static double create_from_options(const double rolloff_location) {
+    return rolloff_location;
+  }
+};
+
+/*!
+ * \brief Rate of roll off for rolling off the shift.
+ */
+struct RolloffRate : db::SimpleTag {
+  using type = double;
+  using option_tags = tmpl::list<OptionTags::RolloffRate>;
+  static constexpr bool pass_metavariables = false;
+  static double create_from_options(const double rolloff_rate) {
+    return rolloff_rate;
+  }
+};
+
+/*!
+ * \brief Rolled-off shift (i.e. the shift used in computing the fluxes).
+ */
+struct RolledOffShift : db::SimpleTag {
+  using type = tnsr::I<DataVector, 3, Frame::Inertial>;
+};
+
+/*!
+ * \brief Pi computed using the rolled-off shift.
+ */
+struct PiWithRolledOffShift : db::SimpleTag {
+  using type = Scalar<DataVector>;
+};
+
+/*!
+ * \brief Action that moves the derivative of the scalar field to a different
+ * tag, for importing into the evolution code.
+ */
+struct MoveDerivToPhi : ::CurvedScalarWave::Tags::Phi<3>, db::ComputeTag {
```

Comment:
Is this only to "rename" the output dataset? No need to do that, this can and should be done on the importing side. Don't even need to output deriv(psi) necessarily because the derivative can be taken on the evolution grid after importing as well (that's what we do for GH).